### DISCUSSION TITLE
operahouse

### DISCUSSION THREAD

**Discussion by giscus** [TEA-UCSD/TEA-UCSD-Website#5] [Announcements]
# operahouse

https://tea.ucsd.edu/operahouse

**Comment by YuktaChidanandan** [comment]
### OPERAHOUSE is still looking for volunteers!
Fill out our [scare acting form](https://docs.google.com/forms/d/e/1FAIpQLSfCNCifN2f0i6uZ0nKpDUiR7f8jYPL068Kvryn-KzpspCl3jg/viewform) (same as above), submit our [volunteer form](https://docs.google.com/forms/d/e/1FAIpQLSc-xr6YfXTiIQiZftAONc4osYkcv89MnqjoXCvNOf93-L4PHA/viewform?usp=sf_link) (operational help), and join our [Discord server](https://discord.gg/PqdAAyR8uX) for more information!

**Comment by YuktaChidanandan** [comment]
### Bloody Valentine: Scream Your Heart Out!
Don't forget to [sign up](https://docs.google.com/forms/d/e/1FAIpQLSfRDuMNWVn59zpEHIyJyk2cXSTVPSN6daSaWQtwiebNxnz_DA/viewform?usp=sf_link) for our scream competition! Kick off the season of horror-romance by winning glory for you and your (broken?) heart. Don't forget to come out on February 10th at 1:00pm to the Price Center Plaza.

**Comment by YuktaChidanandan** [comment]
### [Registration](https://docs.google.com/forms/d/e/1FAIpQLScAATNSo7aUoncPtVoBdHkk7JRuH0S1EFLn6kJ1rPRA6_laaw/viewform) is open!
Both event dates (Friday Feb 17th and Saturday Feb 18th, 6-10pm) are open for registration! One person should submit this form for the whole group (up to 6 people). More detailed information (how to line up, when to arrive for your assigned time slot, and more) will be emailed to the address on file close to the event date!

**Comment by YuktaChidanandan** [comment]
### OPERAHOUSE Final Casting Call!
Sign up [here](https://docs.google.com/forms/d/e/1FAIpQLSfXWX0Giz7W-nxr7ran4odZ6lsEaQPqhigFQNPSFVsMVt-eNA/viewform?usp=sf_link) if you'd like to be involved! This form is DIFFERENT from the interest form above (but if you've already filled out the original interest form, we will reach out to you shortly. Get ready to get spooky!
